exercise3/src/camera_demo/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.8)
+project(camera_demo)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> REQUIRED)
+
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+
+# Integrated cactus-rt dependencies for the workshop
+add_subdirectory(
+  ${CMAKE_CURRENT_SOURCE_DIR}/../../../vendor/cactus-rt-integrated
+  ${CMAKE_CURRENT_BINARY_DIR}/cactus-rt-build
+)
+
+include_directories(
+  include
+  ${OpenCV_INCLUDE_DIRS}
+)
```

Comment:
This doesn't actually do anything, does it?

Reply:
Correct. I'll remove it anyways